lib/cubit/webhook/webhook_state.dart

```diff
@@ -1,29 +1,55 @@
 class WebhookState {
   final String? lnurlPayUrl;
+  final String? lnAddress;
+  final String? lnAddressUsername;
+  final String? webhookError;
+  final String? webhookErrorTitle;
   final String? lnurlPayError;
   final String? lnurlPayErrorTitle;
   final bool isLoading;
 
   WebhookState({
     this.lnurlPayUrl,
+    this.lnAddress,
+    this.lnAddressUsername,
+    this.webhookError,
+    this.webhookErrorTitle,
     this.lnurlPayError,
     this.lnurlPayErrorTitle,
     this.isLoading = false,
   });
 
   WebhookState copyWith({
     String? lnurlPayUrl,
+    String? lnAddress,
```

Comment:
Should we add `lnAddressUsername` in the `copyWith()` and `toString()` also?

Reply:
Re: https://github.com/breez/misty-breez/pull/313#issuecomment-2614849426

Moved `lnAddressUsername` to secure storage to avoid persistence issues between app updates. 71197071097915cfe41f0abec7806d70965627c4